kommunicate/src/main/java/io/kommunicate/services/KmService.java

```diff
@@ -104,12 +106,12 @@ public static Contact getAssigneeContact(Channel channel, BaseContactService con
             String conversationAssignee = null;
             String conversationTitle = null;
 
-            if (metadataMap.containsKey(KommunicateUI.CONVERSATION_ASSIGNEE)) {
-                conversationAssignee = metadataMap.get(KommunicateUI.CONVERSATION_ASSIGNEE);
+            if (metadataMap.containsKey(KmConstants.CONVERSATION_ASSIGNEE)) {
```

Comment:
@reytum is this and below code same as present in getSupportGroupContact ? can we move it to common function and reuse that.

Reply:
Done